src/slices/CustomerStories/CustomerStories.test.tsx

```diff
@@ -1,44 +1,36 @@
 import React from 'react';
 import { render, screen } from '../../test/testUtils';
-import { mergeDeep } from '../../utils/mergeDeep';
 import { strapiCustomerStoryMock } from '../../test/strapiMocks/strapiCustomerStory';
-import { CustomerStoriesProps } from './CustomerStories';
 import CustomerStories from '.';
 
-const defaultProps: CustomerStoriesProps = {
-  slice: {
-    customer_stories: [
-      {
-        ...strapiCustomerStoryMock,
-        attributes: {
-          ...strapiCustomerStoryMock.attributes,
-          createdAt: '2022-01-10T15:04:32.897Z',
-        },
-      },
-    ],
-  },
-  customerStories: [strapiCustomerStoryMock],
-};
+const slice = { customer_stories: [strapiCustomerStoryMock] };
 
-const setup = (props = {}) => {
-  const combinedProps = mergeDeep(defaultProps, props);
-  render(<CustomerStories {...combinedProps} />);
+const setup = () => {
+  render(
+    <CustomerStories
+      slice={slice}
+      customerStories={[strapiCustomerStoryMock]}
+    />
+  );
 };
 
-describe('The CustomerStories component', () => {
-  it('displays all the customer story titles', () => {
+describe('The CustomerCard component', () => {
+  it('displays the customer stories title', () => {
     setup();
 
     expect(
-      screen.getByText(strapiCustomerStoryMock.attributes.title)
+      screen.getByText(strapiCustomerStoryMock.attributes.title || '')
     ).toBeInTheDocument();
   });
 
   it('displays nothing if the customer story that should be displayed does not exists in the delivered customer stories', () => {
-    setup({ customerStories: [] });
+    const slice = {
+      customer_stories: [],
```

Comment:
```suggestion
      customerStories: [],
```
Is there a reason for not using camel case?

Reply:
I don't know the reason why we don't use camel case here, it was done this way before this story. My idea was not to rebuild the concepts. What do you think?